components/frontend/src/widgets/TabPane.js

```diff
@@ -0,0 +1,44 @@
+import "./TabPane.css"
+
+import { bool, element, oneOfType, string } from "prop-types"
+import { useContext } from "react"
+import { Menu } from "semantic-ui-react"
+
+import { DarkMode } from "../context/DarkMode"
+import { Icon, Label, Tab } from "../semantic_ui_react_wrappers"
+
+function FocusableTab({ error, iconName, label }) {
+    const className = useContext(DarkMode) ? "tabbutton inverted" : "tabbutton"
```

Comment:
Can't we use a "darkmode" class instead of "inverted"?

Reply:
Yeah, we could here, but `inverted` is the classname used by Semantic UI CSS, so I wanted to be consistent with that.